crmsh/ui_node.py

```diff
@@ -9,9 +9,99 @@
 from . import utils
 from . import xmlutil
 from .msg import common_err, syntax_err, no_prog_err, common_info, common_warn
+from .msg import common_debug, common_err
 from .cliformat import cli_nvpairs, nvpairs2list
 from . import term
+from .cibconfig import cib_factory
+from .ui_resource import rm_meta_attribute
 
+def update_xml_node(xml_node, attr, value):
+    '''
+    xml_node.attr := value
+
+    Besides, it asks the user if he wants to
+    1) remove both the attr and conflicting_attr
+    in primitives, groups and clones
+    2) remove the conflicting attribute in the node itself
+    '''
+
+    conflicting_attr = ''
+    if 'maintenance' == attr:
+        conflicting_attr = 'is-managed'
+    if 'is-managed' == attr:
+        conflicting_attr = 'maintenance'
+
+    # Override the resources on the node
+    resources = [x.node for x in cib_factory.cib_objects
+        if x.obj_type == "primitive"
+            or x.obj_type == "group"
+            or x.obj_type == "clone"]
```

Comment:
I would prefer this:
`resources = [x.node for x in cib_factory.cib_objects if x.obj_type in ["primitive", "group", "clone"]]`

Reply:
This will really be better.

Reply:
Done.